### DISCUSSION TITLE
Docstring examples not showing when using the Pylance languageserver

### DISCUSSION THREAD

**Discussion by MarcSkovMadsen** [holoviz/hvplot#1568] [General]
hvplot==0.8.2

Unfortunately the examples of the docstrings in hvplot using `..code-block::` do not show in tooltips when using the Pylance language server. They do when using the Jedi language server which I use.

I can see Pandas uses `<<<` for code blocks

![image](https://user-images.githubusercontent.com/42288570/204075763-2fd191c2-4dfb-4367-ba0b-16bf7026a274.png)

They show nicely with both the Pylance and Jedi language server.

Originally I used these. But using `>>>` just made copying of code snippets harder as you first have to remove the `>>>` before you can apply them. And I also thought using `..code-block::` was better for Sphinx documentation.

Is it OK I change `..code-block::` to `>>>` in the documentation @maximlt and others?

**Comment by maximlt** [comment]
I think we need a more systematic approach here. We need to define clearly the environments we want to support. This is roughly going to include IDEs, notebooks, REPLs, and automated documentation. When this set is defined, we'll need to find the format that works best for these environments, maybe we'll have to make some tradeoffs and if so we'll need to clearly identify them. Once this is done we can propagate the convention found across HoloViz.
